src/test/java/seedu/address/testutil/TypicalModuleClass.java

```diff
@@ -13,9 +13,8 @@
 
 public class TypicalModuleClass {
 
-    // TODO: Move to TypicalStudents after UUID has been implemented in Student class.
-    public static final UUID STUDENT_UUID_1 = UUID.fromString("224bdbab-e1b1-4bb1-b34c-bebe9fb17fbc");
-    public static final UUID STUDENT_UUID_2 = UUID.fromString("71c0755a-622c-49e8-b938-693a0dd598dc");
+    public static final UUID STUDENT_UUID_1 = UUID.fromString("0c527a3f-8a6f-4c16-b57d-563a9c43cf6b");
+    public static final UUID STUDENT_UUID_2 = UUID.fromString("5c09bbdc-b89c-482a-a4b0-32ede6d679e7");
```

Comment:
I think you missed out on the TODO comment. Perhaps instead of importing these `STUDENT_UUID_X` in test cases,, we can call `ALICE.getUuid()` from `TypicalStudent` class instead.

Reply:
Thats a good idea

Reply:
Resolved by pulling information from ALICE directly and removing TODO comment